### DISCUSSION TITLE
I restart verdaccio, WebUI show packages are missing , but In filefolder  packages  are still there.

### DISCUSSION THREAD

**Discussion by FzuLiWei** [verdaccio/verdaccio#2816] [Q&A]
**Your Environment**
  
 * **verdaccio version**: 5.3.1
 * **node version** [13.14.0-x64]:
 * **package manager**: [npm 6.14.4]
 * **os**: [windows7]
 * **platform**: [npm]

**Describe the bug**

step 1 : I publish some packages .

![2021 12 20_李炜 5ecbcc12f778631f06dee17dd5060a49](https://user-images.githubusercontent.com/7477767/146733825-42cb7b21-d0f6-4b3c-b1be-bec081c5030c.png)

![2021 12 20_李炜 fa0bcea7dd7d68719ed67133b1170e6b](https://user-images.githubusercontent.com/7477767/146733862-03b3aae6-23a6-4c36-83cb-097f2201cc00.png)

step2 : I use pm2 delete 0 ,

step3 : then pm2 start xxxx\verdaccio. 

I address Ip ,but can not see packages In Web. But In verdaccio\storage is still there I can publish or unpublish normally on cmd.

![1](https://user-images.githubusercontent.com/7477767/146733650-03b6cbc6-fdf5-415a-8648-f1da4eee3c05.png)

![2021 12 20_李炜 decfd3cee3f029edac8f424c7732cefc](https://user-images.githubusercontent.com/7477767/146733693-b60db49e-2e5e-42fe-a224-a9142df58806.png)

**To Reproduce**

**Expected behavior**

**Screenshots, server logs, package manager log**

**Configuration File (cat ~/.config/verdaccio/config.yaml)**
[config.txt](https://github.com/verdaccio/verdaccio/files/7743621/config.txt)

**Environment information**

**Debugging output**

- `$ NODE_DEBUG=request verdaccio` display request calls (verdaccio <--> uplinks)
- `$ DEBUG=verdaccio* verdaccio` enable extreme verdaccio debug mode (verdaccio api)
- `$ npm -ddd` prints:
- `$ npm config get registry` prints:

**Contribute to Verdaccio**

- [ ] I'm willing to fix this bug 🥇

**Comment by wangjieCode** [comment]
Is this problem fixed now?

**Reply by juanpicado** [reply]
I don't see enough eviedence to investigate here, if there were logs, debug etc I would bother but I also use pm2 and I don't loose packages every time I restart, so for me is not a bug, maybe a configuration issue, unless someome prove me wrong I'd be help to investigate further.
